assets/sass/components/_error_summary.scss

```diff
@@ -0,0 +1,27 @@
+$aplus-danger: #A50000;
+$aplus-background: #FFF;
+$width: 100%;
+$padding-amount: 5px;  
+$margin: 20px;
+
+
+.error-summary {
+	> .background {
+		background: repeating-linear-gradient(
+			45deg,
+			$aplus-danger,
+			$aplus-danger $padding-amount,
+			$aplus-background $padding-amount,
+			$aplus-background $padding-amount * 2
+		);
+		padding: $padding-amount;
+		color: $aplus-danger;
+		margin: $margin 0px; 	
+		> .text-container {
```

Comment:
These are pretty short selectors. It's very possible that there is css class that targets `.text-container`. So maybe `&__text` (i.e. `.error-summary__background__text`).